lbcapi/api.py

```diff
@@ -32,7 +35,11 @@ def __init__(self):
         self.hmac_key = None
         self.hmac_secret = None
 
-    def call(self, method, url, params={}, stream=False, files=None):
+    def call(self, method, url, params=None, stream=False, files=None):
```

Comment:
What is wrong with the original solution? In the end it is the same, except the new solution has more complexity, right?